python/py-keep/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-keep
+version             2.10.1
+license             MIT
+supported_archs     noarch
+maintainers         nomaintainer
+
+description         A Meta CLI toolkit
+long_description    A Meta CLI toolkit: your personal shell command keeper
+homepage            https://pypi.org/project/keep/
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            keep-${version}
```

Comment:
again, these `master_sites` and `distname` fields are set to their default value and that's already done in the `python` PG - so no need to repeat that here

Reply:
Thank you. Removed.